### ISSUE TITLE
PQ2

### ISSUE THREAD

**Issue by D4ngerKitty** [LucasMayhew/the-phoenixs-quest-the-cloudy-climb-dlc#1]
@UnsignedArduino Hey I've started work on PQ 2 right now heres a movement test I worked on 
you can help me code if you want but I'm fine working one this by myself 
https://makecode.com/_H3iR4rWUH3VJ

**Comment by UnsignedArduino** [comment]
Hi @LucasMayhew, it's been a while

I'm surprised you decided to make the Pheonix's Quest 2 - I'm so excited for it! I would love to help in any way possible, but due to my schedule, I do not think I would be able to help code it. 

I just tried the move test, and it feels weird that letting go of A after jumping resets the VY like it just hit a wall. Also should up be jump so A can be attack?

**Comment by D4ngerKitty** [comment]
yeah its a test to test out the side on B mostly and things will be changed. 
I am using the platformer extension basic jump so that can be changed in the future as well

**Comment by D4ngerKitty** [comment]
https://makecode.com/_69TgHH9LMfCh updated the link 
worked on a basic attack function that should damage all the enemies you fly though
